docs/Guides/visualizing-mcap-in-foxglove-studio/store-in-foxglove-data-platform.md

```diff
@@ -0,0 +1,9 @@
+# Store in Foxglove Data platform
+
+[Sign up for a Foxglove account ](https://console.foxglove.dev) to access the data management features of [Foxglove Data Platform](https://foxglove.dev/data-platform) .
+
+## Benefits
+
+Instead of having to pass around hard drives every time you want to share data, you can start importing your MCAP data into a central repository for all your teammates to access and collaborate on.
```

Comment:
I think we can make a stronger pitch if we feel this is the place for a DP pitch, but it doesn't make sense to me to pitch paid foxglove products in the mcap repo.

I think there could be a sense in which that works against us/mcap, by suggesting that mcap is only useful if you want to use foxglove - so I would probably remove this guide and maybe relocate it to the website/DP docs, but if we don't want that I can provide some expanded copy.